src/engine/BMGame.php

```diff
@@ -59,46 +59,46 @@
  * @SuppressWarnings(PMD.UnusedPrivateField)
  */
 class BMGame {
-    // properties -- all accessible, but written as private to enable the use of
+    // properties -- all accessible, but written as protected to enable the use of
     //               getters and setters
-    private $gameId;                // game ID number in the database
-    private $playerIdArray;         // array of player IDs
-    private $nPlayers;              // number of players in the game
-    private $roundNumber;           // current round number
-    private $turnNumberInRound;     // current turn number in current round
-    private $activePlayerIdx;       // index of the active player in playerIdxArray
-    private $playerWithInitiativeIdx; // index of the player who won initiative
-    private $buttonArray;           // buttons for all players
-    private $activeDieArrayArray;   // active dice for all players
-    private $attack;                // array('attackerPlayerIdx',
+    protected $gameId;                // game ID number in the database
+    protected $playerIdArray;         // array of player IDs
+    protected $nPlayers;              // number of players in the game
+    protected $roundNumber;           // current round number
+    protected $turnNumberInRound;     // current turn number in current round
+    protected $activePlayerIdx;       // index of the active player in playerIdxArray
+    protected $playerWithInitiativeIdx; // index of the player who won initiative
+    protected $buttonArray;           // buttons for all players
+    protected $activeDieArrayArray;   // active dice for all players
+    protected $attack;                // array('attackerPlayerIdx',
```

Comment:
Why the change of all these variables from private to protected as part of this pull?  I don't see a problem with the change, but i don't understand what it had to do with Warrior.